From 0088c36ad539d0e61f70121f335d970c5b2f5a4a Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Sat, 06 Mar 2021 09:13:05 +0200
Subject: [PATCH] [SW-0] gaudi2_arc_gnu: fix compile error

The below commit had a fix for compilation and it was
not done for gaudi2.
    [SW-32407] boot_if: prepare for comms desc cont.

    The communication descriptor will be added to the
    boot_if and will include types from stdint.h, but
    it can't include the stdint.h by
    itself, so we need to do this for boot_if.
    Add stdint.h and a comment about its requirement.

Change-Id: I7291b2aa995aa96dcb04d88f70f021919177f17f
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arc_gnu/common.c b/gaudi2_arc_gnu/common.c
index 12858f6..41347b4 100644
--- a/gaudi2_arc_gnu/common.c
+++ b/gaudi2_arc_gnu/common.c
@@ -21,7 +21,8 @@
 #include <psoc_reset_conf_regs.h>
 #include <cpu_timestamp_regs.h>
 
-#include <hl_boot_if.h>
+#include <stdint.h>
+#include <hl_boot_if.h>		/* requires stdint.h */
 #include <gaudi2/gaudi2_reg_map.h>
 #include <common/pci_ids.h>
 #include <gaudi2_blocks_linux_driver.h>
